### PR TITLE
Fix dashboard

### LINKED ISSUE #455: Resolve Metric issues, : poller don't get metrics correctly
### Describe the problem you discovered

- Metric should include all txns
- when timeout, metric dashboard should have metrics have of node itself.
image

Also on the dashboard it's seems that the datas are additionned instead of being averaged

### Describe the solution you'd like

Metric poller should retrieve datas without error.
Dashboard view should have accurate values

### PR DESCRIPTION
**Pull request by samuelmanzanera**
# Description

Fix the dashboard metrics aggregation and plotting.

Fixes #455 

## Type of change

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

By running transactions the graphs should be updated with up/down lines.

# Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by apoorv-2204** [comment]
LGTM: Screenshot_20220905_113042

File: assets/js/metric_config.js [hunk 1]
```diff
@@ -304,15 +284,15 @@ function update_live_visuals(metric_obj, points) {
   update_chart_data(metric_obj.archethic_mining_proof_of_work_duration, metric_obj.x_axis_data, points, "archethic_mining_proof_of_work_duration");
   update_chart_data(metric_obj.archethic_mining_full_transaction_validation_duration, metric_obj.x_axis_data, points, "archethic_mining_full_transaction_validation_duration");
 
-  update_guage_data(metric_obj.archethic_p2p_send_message_duration, points, "archethic_p2p_send_message_duration");
```
Comment (apoorv-2204) [inline assets/js/metric_config.js:307]: Unused guage methods can be removed.

File: assets/js/metric_config.js [hunk 2]
```diff
@@ -248,7 +227,7 @@ function update_chart_data(chart_obj, x_axis_data, points, point_name) {
     shifted_value = 0;
```
Comment (apoorv-2204) [inline assets/js/metric_config.js:227]: We can remove ,line 135..242. of guage object

File: lib/archethic/telemetry.ex [hunk 3]
```diff
@@ -41,84 +41,102 @@ defmodule Archethic.Telemetry do
...
+        reporter_options: [
+          buckets: [100, 200, 500, 700, 1000, 1500, 2000, 3000, 5000]
+        ]
       ),
       distribution("archethic.p2p.send_message.duration",
-        unit: {:native, :second},
+        unit: {:native, :millisecond},
         measurement: :duration,
-        reporter_options: [buckets: [0.01, 0.025, 0.05, 0.1, 0.2, 0.5, 0.8, 1.0, 1.25, 1.5, 2.0]],
+        reporter_options: [
```
Comment (apoorv-2204) [inline lib/archethic/telemetry.ex:104]: can we split to more buckets?

**Review by apoorv-2204 — CHANGES_REQUESTED** [review]
Looks nice to me.

File: lib/archethic/telemetry.ex (hunk 3 above)
Comment (samuelmanzanera) [reply lib/archethic/telemetry.ex:104]: Yes we can if we need

File: assets/js/metric_config.js [hunk 4]
```diff
@@ -111,107 +111,6 @@ function generateEchartObjects(heading, echartContainer, x_axis_data) {
...
-  guage_options && guage.setOption(guage_options);
-  window.addEventListener('resize', function () {
-    guage.resize();
-  });
-
-  return { "guage": guage, "max": 0 }
-}
-
 // for proper display of axis labels
 function exponent_formatter(new_point) {
```
Comment (apoorv-2204) [inline assets/js/metric_config.js:115]: this can also be removed

**Pull request closed without merging**